src/libcore/convert.rs

```diff
@@ -50,20 +50,41 @@
 
 use str::FromStr;
 
-/// A cheap, reference-to-reference conversion.
+/// A cheap reference-to-reference conversion. Used to convert a value to a
+/// reference value within generic code.
```

Comment:
Have run the rustdoc build and appears to be all fine. 
![trait_summary](https://cloud.githubusercontent.com/assets/15388206/25106923/9d334410-240f-11e7-953d-de746304e843.png)